lib/uniris/networking/ip_lookup/config.ex

```diff
@@ -0,0 +1,75 @@
+defmodule Uniris.Networking.IPLookup.Config do
+  @moduledoc """
+  Module provides configuration assets for IP Lookup module.
+  """
+
+  # Constants
+
+  @env_hostname "HOSTNAME"
+
+  @error_sys_config "No system config found"
+  @error_ip_provider_invalid "IP provider is invalid"
+  @error_hostname_invalid "Hostname is invalid"
+  @error_hostname_env_missed "#{@env_hostname} env var is mandatory"
+
+  # Public
+
+  @doc """
+  :load_from_system_env config parameter is optional.
+  Parameter absence treats as false
+  """
+  @spec load_from_sys_env?() :: {:ok, keyword()} | {:error, binary}
+  def load_from_sys_env? do
+    with {:ok, config} <- fetch() do
+      Keyword.fetch(config, :load_from_system_env)
+      |> case do
+        {:ok, true} -> {:ok, true}
+        :error -> {:ok, false}
+        {:ok, false} -> {:ok, false}
+      end
+    else
+      {:error, reason} -> {:error, reason}
+    end
+  end
+
+  @spec ip_provider() :: {:ok, module} | {:error, binary}
+  def ip_provider do
```

Comment:
IP provider could be redirected to the IP Lookup module as a configuration implementation switch

Reply:
ok